install/terraform/gcp/variables.tf

```diff
@@ -11,3 +11,9 @@ variable "zone" {
   type    = string
   default = "us-central1-a"
 }
+
+variable "attach_gpu_nodepools" {
```

Comment:
maybe call it create_l4_gpu_nodepools ?

Reply:
Will see what we decide on here: https://github.com/substratusai/substratus/pull/147#discussion_r1279308694